tests/test_level/test_level_02.py

```diff
@@ -557,6 +557,25 @@ def test_sleep(self):
 
         self.multi_level_tester(code=code, expected=expected)
 
+    def test_sleep_micro_bit_argument(self):
+        code = "sleep 2"
+        expected = "sleep(2000)"
+
+        self.multi_level_tester(code=code, expected=expected, max_level=5, microbit=True)
+
+    def test_sleep_micro_bit_no_argument(self):
+        code = "sleep"
+        expected = "sleep(1000)"
+
+        self.multi_level_tester(code=code, expected=expected, max_level=5, microbit=True)\
+
+
+    def test_sleep_micro_bit_variable_argument(self):
+        code = " duration = 2\nsleep duration"
```

Comment:
This isn't a valid Hedy program for 2 reasons: 1) there is a space in the beginning of the program and 2) the `=` should be changed to `is`

Reply:
Great catch didn't even realized it. Thank you.